js/front/dataset/index.js

```diff
@@ -0,0 +1,224 @@
+/**
+ * Dataset display page JS module
+ */
+// Catch all errors
+import 'raven';
+
+// ES6 environment
+import 'babel-polyfill';
+
+import $ from 'jquery'; // Only needed for tooltips :(
```

Comment:
There is `this.$http` too that we can replace with [fetch](https://github.com/github/fetch)